src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs

```diff
@@ -62,8 +62,8 @@ public IActionResult SignOut([FromRoute] string scheme)
                  {
                      RedirectUri = callbackUrl,
                  },
-                CookieAuthenticationDefaults.AuthenticationScheme,
-                scheme);
+                 CookieAuthenticationDefaults.AuthenticationScheme,
```

Comment:
Addressed a warning about indentation

Reply:
Don't worry about explaining these :). I sometimes use Code Maid to apply spacing rules across all open files. https://marketplace.visualstudio.com/items?itemName=SteveCadwallader.CodeMaid